scripts/apps/monitoring/directives/ItemActionsMenu.ts

```diff
@@ -145,7 +143,7 @@ export function ItemActionsMenu(superdesk, activityService, workflowService, arc
                         });
 
                         // actions(except viewing an item) are not allowed for items in legal archive
-                        if (item._type !== 'legal_archive' && !scope.isRelatedItem) {
+                        if (item._type !== 'legal_archive' && !scope.allowedActions) {
```

Comment:
Avoid using `!x` for non-boolean values. Use `x == null` / `x != null` / `x?.length > 0` etc.